### DISCUSSION TITLE
Session Not working [V5]

### DISCUSSION THREAD

**Discussion by pk4all** [adonisjs/core#3450] [Help]
Hi Team,

I am putting some data to session ``` session.put('acls',JSON.stringify(acls)) ```, It's not working if "acls" data amount big.

Any solution for this.

Thanks.

**Comment by kkusu5591** [comment]
I have the same issue. Any tips for solution?

[vidmate](https://vidm.fun/vidmate/)
[mobdro](https://authorityapk.com/mobdro/)

**Comment by thetutlage** [comment]
The maximum size of cookies is 4MB. Use redis driver is size is greater than that https://stackoverflow.com/questions/640938/what-is-the-maximum-size-of-a-web-browsers-cookies-key
